GitHubActionsの修正2

# As Is
<!-- （今の状態を記載） -->
下記エラーが発生している。
```
Run curl "$deploy_url"
curl: (3) URL using bad/illegal format or missing URL
Error: Process completed with exit code 3.
```

設定ファイルを見直したところ、どうやらタイポをしている。
これが原因ではないかと思ったのでタイポを修正する。
# To Be 
<!-- （理想の状態を記載。タスク完了の条件にもなる。） -->
エラーがなくなること。
# アクション
<!-- （改善のための具体的なアクションを記載） -->
- [ ] タイポ修正

--- a/.github/workflows/deproy.yaml
+++ b/.github/workflows/deproy.yaml
@@ -2,7 +2,7 @@ name: Render Deploy
 
 on:
   push:
-#    branches: [main]
+    branches: [main]
 
 jobs:
   deploy:
@@ -12,6 +12,6 @@ jobs:
         # Only run this step if the branch is main
         if: github.ref == 'refs/heads/main'
         env:
-          deploy_url: ${{ secrets.RENDER_DEPLOY_HOOK__URL }}
+          deploy_url: ${{ secrets.RENDER_DEPLOY_HOOK_URL }}
         run: |
           curl "$deploy_url"

